package.json

```diff
@@ -17,16 +20,25 @@
     "mongoose": "^6.2.1",
     "morgan": "^1.10.0",
     "nodemon": "^2.0.15",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-router": "^6.2.1",
+    "react-router-dom": "^6.2.1",
+    "vite": "^2.8.4",
     "winston": "^3.5.1"
   },
   "devDependencies": {
+    "@vitejs/plugin-react": "^1.2.0",
+    "compression": "^1.7.4",
+    "cross-env": "^7.0.3",
```

Comment:
Compression and cross-env are from vite? Are you sure you used `npm create vite@latest src -- --template react` ? It's strange to have compression adn cross-env in there since we are not going to use them.